resources/sshdconfig/src/metadata.rs

```diff
@@ -39,6 +39,12 @@ pub const REPEATABLE_KEYWORDS: [&str; 12] = [
     "subsystem"
 ];
 
+
+pub const SSHD_CONFIG_HEADER: &str = "# This file is managed by the Microsoft DSC sshdconfig resource.";
```

Comment:
Should probably put the full resource type name here.  It may also make sense to include the version of the resource.